app/src/main/java/com/framgia/soundcloud/screen/listtracks/ListtracksModule.java

```diff
@@ -0,0 +1,40 @@
+package com.framgia.soundcloud.screen.listtracks;
+
+import android.app.Activity;
+import android.content.Context;
+import android.support.annotation.NonNull;
+import com.framgia.soundcloud.data.model.Category;
+import com.framgia.soundcloud.data.source.TrackRepository;
+import com.framgia.soundcloud.data.source.remote.TrackRemoteDataSource;
+import dagger.Module;
+import dagger.Provides;
+
+/**
+ * Created by anh on 25/10/2017.
+ */
+@Module
+public class ListtracksModule {
+    private Activity mActivity;
+    private Category mCategory;
+
+    public ListtracksModule(@NonNull Activity activity, Category category) {
+        mActivity = activity;
+        mCategory = category;
+    }
+
+    @Provides
+    public ListtracksContract.ViewModel provideViewModel(ListtracksContract.Presenter presenter,
```

Comment:
2.9 Parameter ordering in methods

In Android, it is common to define methods that take a Context. If you write a method, Context should be the first parameter. And callback always should be the last parameter.
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md